src/components/WwwFrame/TheFooter.vue

```diff
@@ -534,75 +568,45 @@ $footer-separator-color: $light-green;
 	}
 
 	.large-footer {
-		.groups {
-			display: flex;
-			flex-flow: column wrap;
-			justify-content: flex-start;
-			height: rem-calc(413);
-			width: 100%;
-			text-align: left;
-			font-size: 0.875rem;
-
-			@include breakpoint('xlarge') {
-				height: rem-calc(400);
-			}
-
-			.narrow {
-				width: 25%;
-				padding: 0 0.9375rem;
-				min-height: rem-calc(110);
-
-				&:nth-child(even) {
-					padding-top: 1rem;
-				}
-			}
-
-			.wide {
-				width: 50%;
-				padding: 0 0.9375rem;
+		&__disclaimer {
+			color: $magnemite;
+		}
 
-				&:nth-child(even) {
-					padding-top: 1rem;
-				}
-			}
+		&__block {
+			padding: 1rem 0.9375rem;
+			text-align: left;
+		}
 
-			ul {
-				li {
-					line-height: 1.6;
-				}
-			}
+		& .large-footer__single-line-links {
```

Comment:
Can this just be `&__single-line-links`?

Reply:
I tried that but stylelint gave a warning about the order of this vs styling above.  Same with the one below. I tried disabling stylelint for the line with a comment, but that didnt seem to work at all.